### PR TITLE
add opensearch support

### PR DESCRIPTION
**Pull request by ivan-savciuc**
initial OS support, docs and OS ACL's will following in a separate PR's

### CODE REVIEW

File: aiven/resource_opensearch.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	s := serviceCommonSchema()
+	s[ServiceTypeOpensearch] = &schema.Schema{
+		Type:        schema.TypeList,
+		MaxItems:    1,
+		Computed:    true,
+		Description: "Opensearch server provided values",
+		Optional:    true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"kibana_uri": {
```
Comment (mhoffm-aiven) [inline aiven/resource_opensearch.go:20]: is it still kibana in open search?

File: aiven/resource_service.go [hunk 2]
```diff
@@ -1028,6 +1030,7 @@ func copyConnectionInfoFromAPIResponseToTerraform(
 
 	switch serviceType {
 	case "cassandra":
+	case "opensearch":
```
Comment (mhoffm-aiven) [inline aiven/resource_service.go:1033]: probably the kibana_uri missing?

**Review by mhoffm-aiven — CHANGES_REQUESTED** [review]

File: aiven/resource_opensearch.go (hunk 1 above)
Comment (ivan-savciuc) [reply aiven/resource_opensearch.go:20]: indeed no kibana, double checked

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Pull request merged**